src/test/resources/overseas_entity_v_3_2.json

```diff
@@ -813,7 +813,9 @@
           "identification_legal_form" : "string",
           "identification_place_registered" : "string",
           "identification_registration_number" : "string",
-          "date_became_interested_person" : "2022-06-10"
+          "date_became_interested_person" : "2022-06-10",
+          "is_corporate_body_still_involved_in_trust" : true,
```

Comment:
Very minor, but I would just set this value to be `false` here, as then it makes a bit more sense with the 'ceased_date' field also being present.

Reply:
Changed to false.